.github/renovate.json5

```diff
@@ -26,6 +26,15 @@
       "datasourceTemplate": "golang-version",
       "depNameTemplate": "golang"
     }, {
+// We want a PR to bump golangci-lint versions used in the Makefile, taking it
+// from the official Github repository.
+       "fileMatch": ["^Makefile$"],
+      "matchStrings": [
+        "GOLANGCILINT_VERSION = (?<currentValue>.*?)\\n"
+      ],
+      "datasourceTemplate": "github-tags",
+      "depNameTemplate": "golangci/golangci-lint"
+    }, {
```

Comment:
Is it possible to group this, so we get the action and Makefile versions bumped in the same PR?

Reply:
let me check

Reply:
It's working properly now, @negz, see it on my fork https://github.com/phisco/crossplane/pull/152